core/server/data/migrations/versions/3.0/1-add-type-column.js

```diff
@@ -0,0 +1,42 @@
+const common = require('../../../../lib/common');
+const commands = require('../../../schema').commands;
+
+const createLog = type => msg => common.logging[type](msg);
+
+function createColumnMigration({table, column, dbIsInCorrectState, operation}) {
```

Comment:
I see this method came from `1-add-type-column-to-integrations` :wink: Like it! Think the next time we use 'createColumnMigration' in our migrations we can start thinking of extracting it into some kind of 'migrations-utils' :+1:

Reply:
;) you found it! Yeah that sounds good to me :)